src/types.ts

```diff
@@ -10,6 +16,31 @@ export type {
   RuntimeSlackFunctionHandler as SlackFunctionHandler,
 } from "./functions/types.ts";
 
+// ----------------------------------------------------------------------------
+// Runtime Types
+// ----------------------------------------------------------------------------
+
+// deno-lint-ignore no-explicit-any
+type AnyCustomType = CustomType<any, any, any>;
+
+type AnyTypedObjectParameterDefinition = TypedObjectParameterDefinition<
+  // deno-lint-ignore no-explicit-any
+  any,
+  // deno-lint-ignore no-explicit-any
+  any
+>;
```

Comment:
Can we export these from the relevant directories? That'll help us keep this `src/types.ts` file clean for things we want to surface as our recommended API for devs

Reply:
Since `AnyCustomType` `AnyTypedObjectParameterDefinition` are replaced with `ICustomType` and `TypedObjectParameter` I don't think this will be needed